src/app/pages/ordenesdepago/components/set-datoscompromiso/set-datoscompromiso.component.ts

```diff
@@ -48,12 +48,14 @@ export class SetDatoscompromisoComponent implements OnInit, OnDestroy {
   conv: any;
   subscriptionCambios$: any;
   subEntradaAlmacen$: any;
-  entradaAlmacen: any;
   tituloAccion: any;
   subOrdenesPago$: any;
   ordenPago: any;
   subSolicitudesGiro$;
+  subConvenio$: any;
   editable: boolean = true;
+  flagActa: boolean = true;
+  flagRubro: boolean = true;
```

Comment:
Se recomienda mover esto al comienzo del `constructor(...)`
Y en el constructor, revisar si se pueden mover los `this.store.dispatch(...)` al comienzo del `ngOnInit(...)`